[
  {
    "model": "exceptions.aftexception",
    "pk": 1,
    "fields": {
      "creator": 1,
      "modifiedBy": null,
      "created": "2022-08-25T22:58:18.221Z",
      "lastModified": "2023-01-18T19:42:16.850Z",
      "code": 1,
      "service": "traychg",
      "node": 0,
      "robot": 0,
      "traceback": "Traceback (most recent call last):\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/twisted/internet/defer.py\", line 1445, in _inlineCallbacks\n    result = current_context.run(g.send, result)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/emu/emuRobotServ.py\", line 112, in _move\n    ret = yield RobotBehaviorBase._move(self, *args, check_run=True, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_core/twisted/twistutils.py\", line 65, in unwindGenerator\n    return _cancellableInlineCallbacks(gen)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_core/twisted/twistutils.py\", line 76, in _cancellableInlineCallbacks\n    _inlineCallbacks(None, g, status)\n--- <exception caught here> ---\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 761, in _park_sequence\n    yield self._move(\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/emu/emuRobotServ.py\", line 112, in _move\n    ret = yield RobotBehaviorBase._move(self, *args, check_run=True, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/twisted/internet/defer.py\", line 1445, in _inlineCallbacks\n    result = current_context.run(g.send, result)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 358, in _move\n    defer.returnValue(Robot.move(self, *args, **kwargs))\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robot.py\", line 154, in move\n    return MotionControl.move(self, waypts, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/motionctl.py\", line 470, in move\n    self.expose_path(stretched_path, validate=validate)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/motionctl.py\", line 492, in expose_path\n    self.calculate_kinematics(path)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 790, in calculate_kinematics\n    tabws, arm_orgs, arm_vecs, grip_vecs = self.kine.inverse_geom(\n  File \"src/kinematics.pyx\", line 517, in ckinematics.TBotKinematics.inverse_geom\n    \n  File \"src/kinematics.pyx\", line 662, in ckinematics.TBotKinematics.check_carriage_limits\n    \nckinematics.CarriageLimitError: carriage min limit: 460.69999999999993 exceeded: [ 973.943  956.898  935.141  908.674  877.496  841.608  802.461  763.206\n  723.951  684.695  645.44   607.603  574.349  545.75   520.901  499.294\n  480.928  465.804  453.922  445.282  439.883  437.706  438.992  441.493\n  445.164  450.003  456.01   463.187  471.532  480.991  490.73   500.47\n  510.209  519.949  529.688  539.428  549.167  558.907  568.646  578.385\n  588.125  597.864  607.604  617.343  627.083  636.822  646.562  656.301\n  666.041  675.78   685.52   695.259  704.998  714.738  724.477  734.217\n  743.956  753.696  763.435  773.175  782.738  791.252  798.597  804.773\n  809.781  813.62   816.29   817.791  818.179  819.037  821.096  824.354\n  828.813  834.471  841.33   849.388  858.646  868.638  878.638  888.638\n  898.638  908.638  918.638  928.638  938.638  948.638  958.638  968.638\n  978.638  988.638  998.638 1008.638 1018.638 1028.638 1038.638 1048.638\n 1058.638 1068.638 1078.638 1088.638 1098.638 1108.638 1118.638 1128.638\n 1138.638 1148.638 1158.638 1168.638 1178.638 1188.638 1198.638 1208.638\n 1218.638 1228.638 1238.638 1248.638 1258.638 1268.638 1278.638 1288.638\n 1298.638 1308.638 1318.638 1328.638 1338.638 1348.638 1358.638 1368.638\n 1378.638 1388.638 1398.638 1408.637 1418.637 1428.637 1438.637 1448.637\n 1458.637 1468.637 1478.637 1488.637 1498.637 1508.637 1518.637 1528.637\n 1538.637 1548.637 1558.637 1568.637 1578.637 1588.637 1598.637 1608.637\n 1618.637 1628.637 1638.637 1648.637 1658.637 1668.637 1678.637 1688.637\n 1698.637 1708.637 1718.637 1728.637 1738.637 1748.637 1758.637 1768.637\n 1778.637 1788.637 1798.637 1808.637 1818.637 1828.637 1838.637 1848.637\n 1858.453 1867.19  1874.726 1881.062 1886.198 1890.134 1893.339 1896.959\n 1900.808 1904.657 1908.506 1912.355 1916.204 1920.053 1923.902 1927.751\n 1931.6   1935.449 1939.298 1943.147 1946.996 1950.845 1954.694 1958.543\n 1962.392 1966.241 1970.09  1973.939 1977.788 1981.581 1984.971 1987.9\n 1990.367 1991.388 1992.346 1993.46  1994.54  1995.462 1996.225 1996.83\n 1997.276 1997.563 1997.692 1997.7  ]\n",
      "info": "traychgunit.1 Error: No tray detected in collection",
      "timestamp": "2022-05-02T21:27:41.767Z",
      "report": 1,
      "handled": false,
      "primary": true
    }
  },
  {
    "model": "exceptions.aftexception",
    "pk": 2,
    "fields": {
      "creator": 1,
      "modifiedBy": null,
      "created": "2022-08-25T22:58:41.782Z",
      "lastModified": "2023-01-18T19:42:16.852Z",
      "code": 1,
      "service": "traychg",
      "node": 0,
      "robot": 0,
      "traceback": "Traceback (most recent call last):\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/twisted/internet/defer.py\", line 1445, in _inlineCallbacks\n    result = current_context.run(g.send, result)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/emu/emuRobotServ.py\", line 112, in _move\n    ret = yield RobotBehaviorBase._move(self, *args, check_run=True, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_core/twisted/twistutils.py\", line 65, in unwindGenerator\n    return _cancellableInlineCallbacks(gen)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_core/twisted/twistutils.py\", line 76, in _cancellableInlineCallbacks\n    _inlineCallbacks(None, g, status)\n--- <exception caught here> ---\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 761, in _park_sequence\n    yield self._move(\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/emu/emuRobotServ.py\", line 112, in _move\n    ret = yield RobotBehaviorBase._move(self, *args, check_run=True, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/twisted/internet/defer.py\", line 1445, in _inlineCallbacks\n    result = current_context.run(g.send, result)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 358, in _move\n    defer.returnValue(Robot.move(self, *args, **kwargs))\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robot.py\", line 154, in move\n    return MotionControl.move(self, waypts, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/motionctl.py\", line 470, in move\n    self.expose_path(stretched_path, validate=validate)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/motionctl.py\", line 492, in expose_path\n    self.calculate_kinematics(path)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 790, in calculate_kinematics\n    tabws, arm_orgs, arm_vecs, grip_vecs = self.kine.inverse_geom(\n  File \"src/kinematics.pyx\", line 517, in ckinematics.TBotKinematics.inverse_geom\n    \n  File \"src/kinematics.pyx\", line 662, in ckinematics.TBotKinematics.check_carriage_limits\n    \nckinematics.CarriageLimitError: carriage min limit: 460.69999999999993 exceeded: [ 973.943  956.898  935.141  908.674  877.496  841.608  802.461  763.206\n  723.951  684.695  645.44   607.603  574.349  545.75   520.901  499.294\n  480.928  465.804  453.922  445.282  439.883  437.706  438.992  441.493\n  445.164  450.003  456.01   463.187  471.532  480.991  490.73   500.47\n  510.209  519.949  529.688  539.428  549.167  558.907  568.646  578.385\n  588.125  597.864  607.604  617.343  627.083  636.822  646.562  656.301\n  666.041  675.78   685.52   695.259  704.998  714.738  724.477  734.217\n  743.956  753.696  763.435  773.175  782.738  791.252  798.597  804.773\n  809.781  813.62   816.29   817.791  818.179  819.037  821.096  824.354\n  828.813  834.471  841.33   849.388  858.646  868.638  878.638  888.638\n  898.638  908.638  918.638  928.638  938.638  948.638  958.638  968.638\n  978.638  988.638  998.638 1008.638 1018.638 1028.638 1038.638 1048.638\n 1058.638 1068.638 1078.638 1088.638 1098.638 1108.638 1118.638 1128.638\n 1138.638 1148.638 1158.638 1168.638 1178.638 1188.638 1198.638 1208.638\n 1218.638 1228.638 1238.638 1248.638 1258.638 1268.638 1278.638 1288.638\n 1298.638 1308.638 1318.638 1328.638 1338.638 1348.638 1358.638 1368.638\n 1378.638 1388.638 1398.638 1408.637 1418.637 1428.637 1438.637 1448.637\n 1458.637 1468.637 1478.637 1488.637 1498.637 1508.637 1518.637 1528.637\n 1538.637 1548.637 1558.637 1568.637 1578.637 1588.637 1598.637 1608.637\n 1618.637 1628.637 1638.637 1648.637 1658.637 1668.637 1678.637 1688.637\n 1698.637 1708.637 1718.637 1728.637 1738.637 1748.637 1758.637 1768.637\n 1778.637 1788.637 1798.637 1808.637 1818.637 1828.637 1838.637 1848.637\n 1858.453 1867.19  1874.726 1881.062 1886.198 1890.134 1893.339 1896.959\n 1900.808 1904.657 1908.506 1912.355 1916.204 1920.053 1923.902 1927.751\n 1931.6   1935.449 1939.298 1943.147 1946.996 1950.845 1954.694 1958.543\n 1962.392 1966.241 1970.09  1973.939 1977.788 1981.581 1984.971 1987.9\n 1990.367 1991.388 1992.346 1993.46  1994.54  1995.462 1996.225 1996.83\n 1997.276 1997.563 1997.692 1997.7  ]\n",
      "info": "traychgunit.1 Error: No tray detected in collection",
      "timestamp": "2022-05-02T21:27:41.767Z",
      "report": 2,
      "handled": false,
      "primary": true
    }
  },
  {
    "model": "exceptions.aftexception",
    "pk": 3,
    "fields": {
      "creator": 1,
      "modifiedBy": null,
      "created": "2022-08-25T22:58:43.838Z",
      "lastModified": "2023-01-18T19:42:16.854Z",
      "code": 1,
      "service": "traychg",
      "node": 0,
      "robot": 0,
      "traceback": "Traceback (most recent call last):\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/twisted/internet/defer.py\", line 1445, in _inlineCallbacks\n    result = current_context.run(g.send, result)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/emu/emuRobotServ.py\", line 112, in _move\n    ret = yield RobotBehaviorBase._move(self, *args, check_run=True, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_core/twisted/twistutils.py\", line 65, in unwindGenerator\n    return _cancellableInlineCallbacks(gen)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_core/twisted/twistutils.py\", line 76, in _cancellableInlineCallbacks\n    _inlineCallbacks(None, g, status)\n--- <exception caught here> ---\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 761, in _park_sequence\n    yield self._move(\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/emu/emuRobotServ.py\", line 112, in _move\n    ret = yield RobotBehaviorBase._move(self, *args, check_run=True, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/twisted/internet/defer.py\", line 1445, in _inlineCallbacks\n    result = current_context.run(g.send, result)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 358, in _move\n    defer.returnValue(Robot.move(self, *args, **kwargs))\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robot.py\", line 154, in move\n    return MotionControl.move(self, waypts, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/motionctl.py\", line 470, in move\n    self.expose_path(stretched_path, validate=validate)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/motionctl.py\", line 492, in expose_path\n    self.calculate_kinematics(path)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 790, in calculate_kinematics\n    tabws, arm_orgs, arm_vecs, grip_vecs = self.kine.inverse_geom(\n  File \"src/kinematics.pyx\", line 517, in ckinematics.TBotKinematics.inverse_geom\n    \n  File \"src/kinematics.pyx\", line 662, in ckinematics.TBotKinematics.check_carriage_limits\n    \nckinematics.CarriageLimitError: carriage min limit: 460.69999999999993 exceeded: [ 973.943  956.898  935.141  908.674  877.496  841.608  802.461  763.206\n  723.951  684.695  645.44   607.603  574.349  545.75   520.901  499.294\n  480.928  465.804  453.922  445.282  439.883  437.706  438.992  441.493\n  445.164  450.003  456.01   463.187  471.532  480.991  490.73   500.47\n  510.209  519.949  529.688  539.428  549.167  558.907  568.646  578.385\n  588.125  597.864  607.604  617.343  627.083  636.822  646.562  656.301\n  666.041  675.78   685.52   695.259  704.998  714.738  724.477  734.217\n  743.956  753.696  763.435  773.175  782.738  791.252  798.597  804.773\n  809.781  813.62   816.29   817.791  818.179  819.037  821.096  824.354\n  828.813  834.471  841.33   849.388  858.646  868.638  878.638  888.638\n  898.638  908.638  918.638  928.638  938.638  948.638  958.638  968.638\n  978.638  988.638  998.638 1008.638 1018.638 1028.638 1038.638 1048.638\n 1058.638 1068.638 1078.638 1088.638 1098.638 1108.638 1118.638 1128.638\n 1138.638 1148.638 1158.638 1168.638 1178.638 1188.638 1198.638 1208.638\n 1218.638 1228.638 1238.638 1248.638 1258.638 1268.638 1278.638 1288.638\n 1298.638 1308.638 1318.638 1328.638 1338.638 1348.638 1358.638 1368.638\n 1378.638 1388.638 1398.638 1408.637 1418.637 1428.637 1438.637 1448.637\n 1458.637 1468.637 1478.637 1488.637 1498.637 1508.637 1518.637 1528.637\n 1538.637 1548.637 1558.637 1568.637 1578.637 1588.637 1598.637 1608.637\n 1618.637 1628.637 1638.637 1648.637 1658.637 1668.637 1678.637 1688.637\n 1698.637 1708.637 1718.637 1728.637 1738.637 1748.637 1758.637 1768.637\n 1778.637 1788.637 1798.637 1808.637 1818.637 1828.637 1838.637 1848.637\n 1858.453 1867.19  1874.726 1881.062 1886.198 1890.134 1893.339 1896.959\n 1900.808 1904.657 1908.506 1912.355 1916.204 1920.053 1923.902 1927.751\n 1931.6   1935.449 1939.298 1943.147 1946.996 1950.845 1954.694 1958.543\n 1962.392 1966.241 1970.09  1973.939 1977.788 1981.581 1984.971 1987.9\n 1990.367 1991.388 1992.346 1993.46  1994.54  1995.462 1996.225 1996.83\n 1997.276 1997.563 1997.692 1997.7  ]\n",
      "info": "traychgunit.1 Error: No tray detected in collection",
      "timestamp": "2022-05-02T21:27:41.767Z",
      "report": 3,
      "handled": false,
      "primary": true
    }
  },
  {
    "model": "exceptions.aftexception",
    "pk": 4,
    "fields": {
      "creator": 1,
      "modifiedBy": null,
      "created": "2022-08-25T22:59:47.914Z",
      "lastModified": "2023-01-18T19:42:16.855Z",
      "code": 1,
      "service": "traychg",
      "node": 0,
      "robot": 0,
      "traceback": "A very different traceback from the rest of them\n\n\t454545454545+42ABadException",
      "info": "traychgunit.1 Error: No tray detected in collection",
      "timestamp": "2022-05-02T21:27:41.767Z",
      "report": 4,
      "handled": false,
      "primary": true
    }
  },
  {
    "model": "exceptions.aftexception",
    "pk": 5,
    "fields": {
      "creator": 1,
      "modifiedBy": null,
      "created": "2022-08-25T22:59:57.040Z",
      "lastModified": "2023-01-18T19:42:16.856Z",
      "code": 1,
      "service": "traychg",
      "node": 0,
      "robot": 0,
      "traceback": "Traceback (most recent call last):\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/twisted/internet/defer.py\", line 1445, in _inlineCallbacks\n    result = current_context.run(g.send, result)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/emu/emuRobotServ.py\", line 112, in _move\n    ret = yield RobotBehaviorBase._move(self, *args, check_run=True, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_core/twisted/twistutils.py\", line 65, in unwindGenerator\n    return _cancellableInlineCallbacks(gen)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_core/twisted/twistutils.py\", line 76, in _cancellableInlineCallbacks\n    _inlineCallbacks(None, g, status)\n--- <exception caught here> ---\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 761, in _park_sequence\n    yield self._move(\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/emu/emuRobotServ.py\", line 112, in _move\n    ret = yield RobotBehaviorBase._move(self, *args, check_run=True, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/twisted/internet/defer.py\", line 1445, in _inlineCallbacks\n    result = current_context.run(g.send, result)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 358, in _move\n    defer.returnValue(Robot.move(self, *args, **kwargs))\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robot.py\", line 154, in move\n    return MotionControl.move(self, waypts, **kwargs)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/motionctl.py\", line 470, in move\n    self.expose_path(stretched_path, validate=validate)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/motionctl.py\", line 492, in expose_path\n    self.calculate_kinematics(path)\n  File \"/home/john/strawberry/venv/lib/python3.8/site-packages/aft_tbot/motion/robotbehavior.py\", line 790, in calculate_kinematics\n    tabws, arm_orgs, arm_vecs, grip_vecs = self.kine.inverse_geom(\n  File \"src/kinematics.pyx\", line 517, in ckinematics.TBotKinematics.inverse_geom\n    \n  File \"src/kinematics.pyx\", line 662, in ckinematics.TBotKinematics.check_carriage_limits\n    \nckinematics.CarriageLimitError: carriage min limit: 460.69999999999993 exceeded: [ 973.943  956.898  935.141  908.674  877.496  841.608  802.461  763.206\n  723.951  684.695  645.44   607.603  574.349  545.75   520.901  499.294\n  480.928  465.804  453.922  445.282  439.883  437.706  438.992  441.493\n  445.164  450.003  456.01   463.187  471.532  480.991  490.73   500.47\n  510.209  519.949  529.688  539.428  549.167  558.907  568.646  578.385\n  588.125  597.864  607.604  617.343  627.083  636.822  646.562  656.301\n  666.041  675.78   685.52   695.259  704.998  714.738  724.477  734.217\n  743.956  753.696  763.435  773.175  782.738  791.252  798.597  804.773\n  809.781  813.62   816.29   817.791  818.179  819.037  821.096  824.354\n  828.813  834.471  841.33   849.388  858.646  868.638  878.638  888.638\n  898.638  908.638  918.638  928.638  938.638  948.638  958.638  968.638\n  978.638  988.638  998.638 1008.638 1018.638 1028.638 1038.638 1048.638\n 1058.638 1068.638 1078.638 1088.638 1098.638 1108.638 1118.638 1128.638\n 1138.638 1148.638 1158.638 1168.638 1178.638 1188.638 1198.638 1208.638\n 1218.638 1228.638 1238.638 1248.638 1258.638 1268.638 1278.638 1288.638\n 1298.638 1308.638 1318.638 1328.638 1338.638 1348.638 1358.638 1368.638\n 1378.638 1388.638 1398.638 1408.637 1418.637 1428.637 1438.637 1448.637\n 1458.637 1468.637 1478.637 1488.637 1498.637 1508.637 1518.637 1528.637\n 1538.637 1548.637 1558.637 1568.637 1578.637 1588.637 1598.637 1608.637\n 1618.637 1628.637 1638.637 1648.637 1658.637 1668.637 1678.637 1688.637\n 1698.637 1708.637 1718.637 1728.637 1738.637 1748.637 1758.637 1768.637\n 1778.637 1788.637 1798.637 1808.637 1818.637 1828.637 1838.637 1848.637\n 1858.453 1867.19  1874.726 1881.062 1886.198 1890.134 1893.339 1896.959\n 1900.808 1904.657 1908.506 1912.355 1916.204 1920.053 1923.902 1927.751\n 1931.6   1935.449 1939.298 1943.147 1946.996 1950.845 1954.694 1958.543\n 1962.392 1966.241 1970.09  1973.939 1977.788 1981.581 1984.971 1987.9\n 1990.367 1991.388 1992.346 1993.46  1994.54  1995.462 1996.225 1996.83\n 1997.276 1997.563 1997.692 1997.7  ]\n",
      "info": "traychgunit.1 Error: No tray detected in collection",
      "timestamp": "2022-05-02T21:27:41.767Z",
      "report": 5,
      "handled": false,
      "primary": true
    }
  },
  {
    "model": "exceptions.aftexception",
    "pk": 6,
    "fields": {
      "creator": 1,
      "modifiedBy": null,
      "created": "2022-10-14T16:18:26.657Z",
      "lastModified": "2023-01-18T19:42:16.857Z",
      "code": 2,
      "service": "drivesys",
      "node": 0,
      "robot": 0,
      "traceback": "",
      "info": "Sevcon Controller Error",
      "timestamp": "2022-09-20T03:56:40.896Z",
      "report": 6,
      "handled": true,
      "primary": true
    }
  },
  {
    "model": "exceptions.aftexception",
    "pk": 7,
    "fields": {
      "creator": 1,
      "modifiedBy": null,
      "created": "2022-10-14T16:18:26.663Z",
      "lastModified": "2022-10-14T16:18:26.663Z",
      "code": 2,
      "service": "harvester",
      "node": 0,
      "robot": 0,
      "traceback": "",
      "info": "Sevcon Controller Error",
      "timestamp": "2022-09-20T03:56:40.896Z",
      "report": 6,
      "handled": false,
      "primary": false
    }
  }
]
